mcm/HTML/requests.html

```diff
@@ -389,7 +392,7 @@
                 <span class="label label-info" ng-show="data[column.db_name]" title="Kegabytes">kB<span>
               </div>
               <div ng-switch-when="time_event" align="center">
-                {{data[column.db_name]}}
+                {{data[column.db_name] | number:4}}
```

Comment:
```suggestion
                {{data[column.db_name]}}
```
This attribute is a list with one number, e.g `[45.78333840332278]`. Including ` | number:4` ends up in an undefined behavior and it is not shown